src/libraries/Common/src/Internal/Cryptography/AsymmetricAlgorithmHelpers.Hash.cs

```diff
@@ -21,10 +23,41 @@ public static byte[] HashData(byte[] data, int offset, int count, HashAlgorithmN
             Debug.Assert(offset >= 0 && offset <= data.Length - count);
             Debug.Assert(!string.IsNullOrEmpty(hashAlgorithm.Name));
 
+#if NET5_0_OR_GREATER
```

Comment:
What project is this file used in that requires the else case?

Reply:
The S.S.C.OpenSsl package has a `netcoreapp3.0` and `net47` TFM:

https://github.com/dotnet/runtime/blob/ca1a5cf09bc2e286c24029f205d376831af1e383/src/libraries/System.Security.Cryptography.OpenSsl/src/System.Security.Cryptography.OpenSsl.csproj#L4